### PR TITLE
Added crud tests interface and sketch of UserDaoTests

### PR DESCRIPTION
**Pull request by vprusa**
Added crud tests interface and sketch of UserDaoTests

### CODE REVIEW

File: src/test/java/cz/muni/fi/pa165/skupina06/team02/rms/app/entities/entity/UserDaoTests.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+import cz.muni.fi.pa165.skupina06.team02.rms.app.dao.UserDao;
+import cz.muni.fi.pa165.skupina06.team02.rms.app.entity.User;
+
+/**
+ * @author Vojtech Prusa
+ */
+@ContextConfiguration(classes = PersistenceSampleApplicationContext.class)
+@TestExecutionListeners(TransactionalTestExecutionListener.class)
+@Transactional
+public class UserDaoTests extends AbstractTestNGSpringContextTests implements UserDao {
```
Comment (vprusa) [inline src/test/java/cz/muni/fi/pa165/skupina06/team02/rms/app/entities/entity/UserDaoTests.java]: tento implements **Dao?

File: src/test/java/cz/muni/fi/pa165/skupina06/team02/rms/app/entities/entity/CRUDTests.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+     * Update test
+     */
+    public void updateTest();
+
+    /**
+     * Delete test
+     */
+    public void deleteTest();
+
+}
```
Comment (433769) [inline src/test/java/cz/muni/fi/pa165/skupina06/team02/rms/app/entities/entity/CRUDTests.java]: Unnecessary. Remove this.

File: src/test/java/cz/muni/fi/pa165/skupina06/team02/rms/app/entities/entity/UserDaoTests.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+import cz.muni.fi.pa165.skupina06.team02.rms.app.PersistenceSampleApplicationContext;
+import cz.muni.fi.pa165.skupina06.team02.rms.app.entity.User;
+
+/**
+ * @author Vojtech Prusa
+ */
+@ContextConfiguration(classes = PersistenceSampleApplicationContext.class)
+public class UserDaoTests extends AbstractTestNGSpringContextTests implements CRUDTests {
+
```
Comment (433769) [inline src/test/java/cz/muni/fi/pa165/skupina06/team02/rms/app/entities/entity/UserDaoTests.java]: The `implements UserDao` is better for checking all use-cases

**Comment by vprusa** [comment]
All solved / fixed, closing

**Pull request closed without merging**